sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/batching/IndexBatchResponse.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.implementation.batching;
+
+import com.azure.search.documents.models.IndexingResult;
+
+import java.util.List;
+
+/**
+ * Model class which keeps track of the service results, the offset from the initial request set if it was split,
+ * and whether the response is an error status.
+ */
+final class IndexBatchResponse {
+    private final int statusCode;
+    private final List<IndexingResult> results;
```

Comment:
List<IndexingResult> or Iterable<IndexingResult>?

Reply:
This is internal only, so I'm going to use List for convenience